libraries/AP_Scripting/lua_scripts.cpp

```diff
@@ -431,6 +434,23 @@ void lua_scripts::run(void) {
             lua_gc(L, LUA_GCCOLLECT, 0);
 
         } else {
+            if (reload == RELOAD_TRUE) {
+                reload = RELOAD_DELAY;
+                hal.scheduler->delay(10000);
+                continue;
+            } else if (reload == RELOAD_DELAY) {
```

Comment:
RELOAD_TRUE implies RELOAD_DELAY here. Why not just step straight into this same block?

Reply:
yeah, your right, I misunderstood how the delays work. Dropped the enum for a bool